### DISCUSSION TITLE
Missing the necessary libraries in the sample test evaluation

### DISCUSSION THREAD

**Discussion by ali-ihsan-gullu** [christianbirchler-org/sdc-testing-competition#9] [General]
When the evaluator-container is run from sample-tools with the command ( $ sudo docker run --rm --name evaluator-container -t evaluator-image -u host.docker.internal:4545), numpy and shapely errors are present. 

The Dockerfile in the "evaluator" needs to include the lines:

RUN pip install numpy
RUN pip install shapely

**Comment by ChristianBirchler** [comment]
Good catch @vatozZ ! Thanks a lot. I will open an issue for this.

**Comment by ChristianBirchler** [comment]
Fixed in #11 .
